iojs.org/aliases.json

```diff
@@ -158,7 +159,8 @@
       "ljharb@gmail.com",
       "othiym23@gmail.com",
       "ryanharrisonlewis@gmail.com",
-      "shelley.vohr@gmail.com"
+      "shelley.vohr@gmail.com",
+      "rtrott@gmail.com"
```

Comment:
you could alphabetize, looks like the others are

Reply:
Ha! I thought I did. Whoops. Will fix.